lerna.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I found yarn to perform **much** better for monorepo installations, with npm lerna has to enter each package and run npm commands in each. yarn has monorepo support builtin so everything happens automatically within a single command. When I've switched to yarn in redux-saga my install times has dropped quite a bit

Reply:
Good point, I think we could migrate to yarn in another PR, but I need to discuss it with the DS team.

Reply:
Yup, we can give it a try since our build and installation time is large. I don't see any cons of using yarn here.